packages/ui/src/proposals/constants/styles.ts

```diff
@@ -5,8 +5,8 @@ import { ListHeaders, ListHeader } from '@/common/components/List/ListHeader'
 export const ProposalColLayout = '296px 148px 156px 60px 92px 1fr'
 
 export const ProposalsListHeaders = styled(ListHeaders)`
-  padding-right: 16px;
-  grid-column-gap: 36px;
+  padding-right: 0px;
+  grid-column-gap: 21px;
```

Comment:
This new layout looks even more cramped on bigger screen:
![image](https://user-images.githubusercontent.com/6571453/218818957-219cd5ed-afed-4b11-93b3-d59eb9786769.png)

Instead I'm proposing to just remove some unused columns from the `ProposalColLayout`. It should be fine because the current proposal tabs has 3 columns and the past proposal past has 4 of them. But `ProposalColLayout` defines 5 fixed width columns + a flexible width one. Deleting the last `92px` wide columns should be enough (actually it should give even more space than removing the right paddings when accounting for the gap between columns) and it will keep the same layout on bigger screens:
```ts
export const ProposalColLayout = '296px 148px 156px 60px 1fr'
```

Reply:
That works as well, i have made the necessary changes,